packages/nimble-components/src/label-provider/table/label-token-defaults.ts

```diff
@@ -7,5 +7,10 @@ export const tableLabelDefaults: { readonly [key in TokenName]: string } = {
     tableGroupExpandLabel: 'Expand group',
     tableGroupsCollapseAllLabel: 'Collapse all groups',
     tableCellActionMenuLabel: 'Options',
-    tableColumnHeaderGroupedIndicatorLabel: 'Grouped'
+    tableColumnHeaderGroupedIndicatorLabel: 'Grouped',
+    tableColumnHeaderSortedAscendingIndicatorLabel: 'Sorted ascending',
+    tableColumnHeaderSortedDescendingIndicatorLabel: 'Sorted descending',
+    tableSelectAllLabel: 'Select all rows',
+    tableGroupSelectAllLabel: 'Select all rows in group',
+    tableRowSelectLabel: 'Select row'
```

Comment:
I wasn't quite sure on all these "Select ..." labels. I thought about making the text something like "Toggle selection of ..." but I didn't think that would read well with a screen reader. My guess is that it would say something like "Toggle selection of all rows, checked", and it seemed confusing to me to saying that toggling something was checked.

I'm open to other suggestions here.

Reply:
I think these labels make sense.

Based on these APG doc pages for [buttons](https://www.w3.org/WAI/ARIA/apg/patterns/button/)/ [switches](https://www.w3.org/WAI/ARIA/apg/patterns/switch/)/ [checkboxes](https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/), I think the label should describe the action/command, and the label should make sense regardless of checked state (since the text shouldn't change based on checked/unchecked). As you said adding 'Toggle' when that would be combined with 'unchecked'/'checked' would be redundant/confusing.